web/client/components/misc/cardgrids/SideCard.jsx

```diff
@@ -72,24 +74,29 @@ module.exports = ({
             </div>}
             <div className="mapstore-side-card-container">
                 <div className="mapstore-side-card-inner">
-                    <div className="mapstore-side-card-info">
-                        {title && <div className="mapstore-side-card-title">
-                            <span>{title}</span>
-                        </div>}
-                        {description && <div className="mapstore-side-card-desc">
-                            <span>{description}</span>
-                        </div>}
-                        {caption && <div className="mapstore-side-card-caption">
-                            <span>{caption}</span>
-                        </div>}
+                    <div className="mapstore-side-card-left-container">
+                        <div className="mapstore-side-card-info">
+                            {title && <div className="mapstore-side-card-title">
+                                <span>{title}</span>
+                            </div>}
+                            {description && <div className="mapstore-side-card-desc">
+                                <span>{description}</span>
+                            </div>}
+                            {caption && <div className="mapstore-side-card-caption">
+                                <span>{caption}</span>
```

Comment:
![image](https://user-images.githubusercontent.com/1279510/78809299-fd659580-79c6-11ea-9f8a-958b0177db6d.png)
I should use the caption entry.